src/core/localize.rkt

```diff
@@ -164,52 +177,84 @@
     (define exacts (list->vector (apply subexprs-fn pt)))
     (define actuals (apply actual-value-fn pt))
 
-    (for ([expr (in-list exprs-list)]
+    (for ([spec (in-vector spec-vec)]
+          [expr (in-list exprs-list)]
           [root (in-vector roots)]
           [exact (in-vector exacts)]
           [actual (in-vector actuals)]
           [expr-idx (in-naturals)])
-      (define err
+      (define true-err
+        (if true-err? ;; ??? Whats the default values for true error literal, variable approx and if?
+            (match (vector-ref nodes root)
+              [(? literal?) 0]
+              [(? variable?) 0]
+              [(approx aprx-spec impl) exact] ;; TODO not sure what to do here.
+              [`(if ,c ,ift ,iff) 0]
```

Comment:
Isn't true error also possible here? Like, shouldn't it be the same as the true error in `ift` and `iff`?